anu_lms.module

```diff
@@ -56,3 +57,84 @@ function anu_lms_node_delete(NodeInterface $node) {
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ *
+ * Enables progress for courses that belong to a category with locking enabled.
+ */
+function anu_lms_node_presave(NodeInterface $node) {
+  if ($node->getType() == 'course') {
```

Comment:
you could turn that check around and return if it doesn't have the sequence value. that's called a guard clause (i saw you are using it in the for loops for counting completed lessons:

```php
if ($node->getType() != 'course') {
  return;
}
```

Reply:
Done